app/utils/mention.ts

```diff
@@ -27,6 +27,16 @@ export const isURLMentionable = ({
       );
     }
 
+    case IntegrationService.Linear: {
+      const settings =
+        integration.settings as IntegrationSettings<IntegrationType.Embed>;
+
+      return (
+        hostname === "linear.app" &&
+        settings.linear?.workspace.name === pathParts[1] // ensure installed workspace name matches with the provided url.
```

Comment:
This doesn't work unfortunately, we'll need to store the workspace `key` and compare that against the first url part